Jenkinsfile-publish-image

```diff
@@ -0,0 +1,79 @@
+pipeline {
+    agent {
+        label 'master'
+    }
+    options {
+        disableConcurrentBuilds()
+    }
+
+    environment {
+        DHIS2_VERSION = ''
+        DOCKER_HUB_REPOSITORY = "${DOCKER_HUB_OWNER}/core"
+        DOCKER_IMAGE_TAG = ''
+        DOCKER_IMAGE_FULL_NAME = ''
+        COMPOSE_PARAMETER = ''
+        TESTS_IMAGE_FULL_NAME = ''
+    }
+
+    stages {
+        stage('Build image') {
+            steps {
+                script {
+                    pom = readMavenPom file: 'dhis-2/pom.xml'
+
+                    DHIS2_VERSION = pom.version.toLowerCase()
+                    echo "DHIS2 version: ${DHIS2_VERSION}"
+
+                    COMPOSE_PARAMETER = "${env.JOB_NAME}"
+                    echo "Docker compose parameter: ${COMPOSE_PARAMETER}"
+
+                    DOCKER_IMAGE_TAG = "${DHIS2_VERSION}-alpine"
```

Comment:
@vilkg I think this might accidently push to many builds if, for instance, a PR or branch has the same `pom.version` but isn't the "latest" of that version (releases being most important, but also snapshots) and might be accidentally pushed to docker hub here with the `2.32.0-alpine` or `2.32-SNAPSHOT-alpine` tag.  Am I missing something?

Reply:
This intends to push only images for master. When we are dealing with PR´s, and if, we want to have images of PR´s, we can tag it as commit/PR reference.